lib/rex/post/meterpreter/extensions/stdapi/fs/file.rb

```diff
@@ -75,7 +75,7 @@ class << self
   #
   # Raises a RequestError if +root+ is not a directory.
   #
-  def File.search( root=nil, glob="*.*", recurse=true, timeout=-1 )
+  def File.search( root=nil, glob="*.*", recurse=true, timeout=-1, startDate:nil, endDate:nil)
```

Comment:
Same comment here regarding making the names more explicit so we can expand on it in the future. These should probably be `modifiedStartDate` and `modifiedEndDate`.

Reply:
updated to m_startDate and m_endDate